### ISSUE TITLE
Error installing r-irkernel

### ISSUE THREAD

**Issue by DavidMertz** [conda/conda#1442]
(r)506-~ % conda install -c r r-irkernel
Fetching package metadata: ........
Solving package specifications: .............
Package plan for installation in environment /Users/dmertz/anaconda/envs/r:

The following NEW packages will be INSTALLED:

    gmp:              5.1.3-0
    ipython:          3.2.1-py34_0
    ipython-notebook: 3.2.1-py34_0
    libxml2:          2.9.2-1
    mistune:          0.6-py34_0
    pip:              7.1.0-py34_0
    python.app:       1.2-py34_4
    pyzmq:            14.7.0-py34_0
    r:                3.2.0-0
    r-base:           3.2.0-0
    r-base64enc:      0.1_2-r3.2.0_1
    r-boot:           1.3_17-r3.2.0_0
    r-class:          7.3_13-r3.2.0_0
    r-cluster:        2.0.2-r3.2.0_0
    r-codetools:      0.2_11-r3.2.0_0
    r-digest:         0.6.8-2
    r-evaluate:       0.7-r3.2.0_1
    r-foreign:        0.8_65-r3.2.0_0
    r-irdisplay:      0.3-r3.2.0_0
    r-irkernel:       0.4-r3.2.0_0
    r-jsonlite:       0.9.16-r3.2.0_0
    r-kernsmooth:     2.23_15-r3.2.0_0
    r-lattice:        0.20_31-r3.2.0_0
    r-magrittr:       1.5-r3.2.0_2
    r-mass:           7.3_42-r3.2.0_0
    r-repr:           0.2-r3.2.0_0
    r-rzmq:           0.7.7-r3.2.0_1
    r-stringi:        0.5_5-r3.2.0_0
    r-stringr:        1.0.0-r3.2.0_0
    r-uuid:           0.1_1-r3.2.0_1

The following packages will be UPDATED:

    r-matrix:         1.1_5-r2.15.2_2  --> 1.2_1-r3.2.0_1
    r-mgcv:           1.8_4-r2.14.0_2  --> 1.8_6-r3.2.0_1
    r-nlme:           3.1_119-r3.0.0_2 --> 3.1_121-r3.2.0_0
    r-nnet:           7.3_8-r2.14.0_2  --> 7.3_10-r3.2.0_0
    r-recommended:    3.1.3-0          --> 3.2.0-r3.2.0_0
    r-rpart:          4.1_8-r2.15.0_2  --> 4.1_10-r3.2.0_0
    r-spatial:        7.3_8-r3.0.0_2   --> 7.3_10-r3.2.0_0
    r-survival:       2.37_7-1         --> 2.38_2-r3.2.0_0
    setuptools:       17.0-py34_0      --> 18.0.1-py34_0
    zeromq:           4.0.5-0          --> 4.0.5-1
    zlib:             1.2.8-0          --> 1.2.8-1

Proceed ([y]/n)? y

Unlinking packages ...
An unexpected error has occurred, please consider sending the             |  18%
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.

Traceback (most recent call last):
  File "/Users/dmertz/anaconda/envs/r/bin/conda", line 5, in <module>
    sys.exit(main())
  File "/Users/dmertz/anaconda/lib/python3.4/site-packages/conda/cli/main.py", line 201, in main
    args_func(args, p)
  File "/Users/dmertz/anaconda/lib/python3.4/site-packages/conda/cli/main.py", line 208, in args_func
    args.func(args, p)
  File "/Users/dmertz/anaconda/lib/python3.4/site-packages/conda/cli/common.py", line 612, in inner
    return func(args, parser)
  File "/Users/dmertz/anaconda/lib/python3.4/site-packages/conda/cli/main_install.py", line 46, in execute
    install.install(args, parser, 'install')
  File "/Users/dmertz/anaconda/lib/python3.4/site-packages/conda/cli/install.py", line 413, in install
    plan.execute_actions(actions, index, verbose=not args.quiet)
  File "/Users/dmertz/anaconda/lib/python3.4/site-packages/conda/plan.py", line 527, in execute_actions
    inst.execute_instructions(plan, index, verbose)
  File "/Users/dmertz/anaconda/lib/python3.4/site-packages/conda/instructions.py", line 148, in execute_instructions
    cmd(state, arg)
  File "/Users/dmertz/anaconda/lib/python3.4/site-packages/conda/instructions.py", line 95, in UNLINK_CMD
    install.unlink(state['prefix'], arg)
  File "/Users/dmertz/anaconda/lib/python3.4/site-packages/conda/install.py", line 671, in unlink
    with open(meta_path) as fi:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/dmertz/anaconda/envs/r/conda-meta/r-nlme-3.1_119-r3.0.0_2.json'
(r)507-~ % conda info
Current conda install:

             platform : osx-64
        conda version : 3.14.1
  conda-build version : 1.11.0
       python version : 3.4.3.final.0
     requests version : 2.7.0
     root environment : /Users/dmertz/anaconda  (writable)
  default environment : /Users/dmertz/anaconda/envs/r
     envs directories : /Users/dmertz/anaconda/envs
        package cache : /Users/dmertz/anaconda/pkgs
         channel URLs : https://conda.anaconda.org/r/osx-64/
                        https://conda.anaconda.org/r/noarch/
                        https://repo.continuum.io/pkgs/free/osx-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/osx-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : /Users/dmertz/.condarc
    is foreign system : False

**Comment by DavidMertz** [comment]
OK, I made an error.  I thought I had setup an initial R basic package in the environment, but I had not on the current machine (apparently I just created an empty environment).  If I first run:

  conda install -c r r

Then follow with:

  conda install -c r r-irkernel

Both installs complete successfully.  Still, the error reporting here feels deficient, and I'm not sure why the dependency isn't simply installed automatically.
